.github/workflows/build-images.yml

```
@@ -0,0 +1,95 @@
+name: Build images
+
+on:
+  - push
+
+jobs:
+  cc7_lcg95:
+    runs-on: ubuntu-latest
+    needs: base_images
+    steps:
+      - uses: actions/checkout@v1
+      - name: Extract branch name
+        shell: bash
+        run: echo "::set-env name=branch::$(echo ${GITHUB_REF#refs/heads/} | sed 's/\//_/g')"
+      - uses: acts-project/Publish-Docker-Github-Action@master
+        with:
+          registry: docker.pkg.github.com
+          name: acts-project/machines/cc7_lcg95
+          username: ${s github.actor  }}
```

```suggestion
          username: ${{ github.actor  }}
```
I think I found the bug.

---

Right that's definitely wrong :D but the build fails before that. This uses the base image from another job, the ubuntu build fails on it's own :/